.travis.yml

```diff
@@ -11,37 +16,28 @@ env:
   global:
     - PATH=/opt/python/3.6.7/bin:$PATH
 
-# Install python3 and utilities dependencies
-addons:
-  apt:
-    packages:
-      - "python3"
-      - "python3-pip"
+language: go
+go:
+- 1.15.x
+go_import_path: github.com/operator-framework/operator-sdk
 
 cache:
   directories:
-    - $HOME/.cache/go-build
-
-go:
-- 1.15.x
+    - ${HOME}/.cache/go-build
+    - $(go env GOPATH)/pkg/mod
```

Comment:
Just double checking two things:
1. Does the `$()` style shell expansion work in this field?
2. I feel like we've gone back and forth on what we cache and don't cache. Is this directory one of the one's we used to cache and then stopped caching?

Reply:
1. [Yes](https://travis-ci.org/github/operator-framework/operator-sdk/jobs/737476890#L240).
1. I don't remember us caching modules, but this does speed up builds significantly. One reason not to do this is we may not catch module issues that affects a user until a non-cached build runs. Can you think of others?

Reply:
1. :+1: TIL!
2. That's the only reason I can think of, and I don't think it outweighs the speedup benefit.